src/System.IO.FileSystem/src/System.IO.FileSystem.csproj

```diff
@@ -76,9 +76,16 @@
     <Compile Include="$(CommonPath)\System\IO\PathInternal.CaseSensitivity.cs">
       <Link>Common\System\IO\PathInternal.CaseSensitivity.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\System\Threading\Tasks\TaskToApm.cs">
+      <Link>Common\System\Threading\Tasks\TaskToApm.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <!-- .NET Standard 1.7 -->
+  <ItemGroup Condition="'$(PackageTargetFramework)' == 'netstandard1.7' and '$(TargetGroup)' != 'net46' and '$(TargetGroup)' != 'net463'" >
```

Comment:
I'm uncomfortable with blank having meaning. It's not immediately obvious. I'd be happier if `TargetGroup` defaulted to `latest`, `latestnetstandard` or `current` perhaps.  Maybe add another property if TargetGroup is blank that we can make clearer conditions off of?

In any case, the file is explicitly 1.7 (or higher). How do we reflect that if we have to continue/reintroduce compiling legacy targets?